dll/win32/shell32/dialogs/folder_options.cpp

```diff
@@ -266,10 +275,34 @@ ShowFolderOptionsDialog(HWND hWnd, HINSTANCE hInst)
     pinfo.pszIcon = MAKEINTRESOURCEW(IDI_SHELL_FOLDER_OPTIONS);
     pinfo.pszCaption = szOptions;
     pinfo.pfnCallback = PropSheetProc;
+    pinfo.nStartPage = (UINT)param;
```

Comment:
../src/dll/win32/shell32/dialogs/folder_options.cpp:278:30: error: cast from 'LPVOID' {aka 'void*'} to 'UINT' {aka 'unsigned int'} loses precision [-fpermissive]